WordPress/Classes/Services/MediaImportService.swift

```diff
@@ -30,13 +30,15 @@ open class MediaExportService: LocalCoreDataService {
 
     // MARK: - Instance methods
 
-    /// Creates a Media object with an absoluteLocalURL for a PHAsset's data, asynchronously.
-    /// - paramater blog: a blog object to where the media object will be added to.
-    /// - paramater post: an optional post object to where the media object will be attached to.
+    /// Imports media from a PHAsset to the Media object, asynchronously.
+    ///
+    /// - paramater asset: the PHAsset media where data will be read from.
+    /// - paramater media: the media object to where media will be imported to.
     /// - parameter onCompletion: Called if the Media was successfully created and the asset's data exported to an absoluteLocalURL.
     /// - parameter onError: Called if an error was encountered during creation, error convertible to NSError with a localized description.
     ///
-    public func exportMediaWith(blog: Blog, post: AbstractPost?, asset: PHAsset, onCompletion: @escaping MediaCompletion, onError: @escaping OnError) {
+    @objc(importAsset:toMedia:onCompletion:onError:)
+    public func `import`(_ asset: PHAsset, to media: Media, onCompletion: @escaping MediaCompletion, onError: @escaping OnError) {
         exportQueue.async {
 
             let exporter = MediaAssetExporter()
```

Comment:
We should rename all of the Exporters to Importers too.

```
let importer = MediaAssetImporter()
importer.importData(forAsset:...
```

Reply:
This ones I actually think they should be exporter, because they export from their original format to a file.